src/clients/meta/MetaClient.cpp

```diff
@@ -2430,6 +2434,18 @@ folly::Future<StatusOr<bool>> MetaClient::heartbeat() {
     }
   }
 
+  // info used in the agent, only set once
+  if (!dirInfoReported_) {
```

Comment:
if the dirInfo is changed, like add a disk, is there need update the info and report?

Reply:
I'll add a TODO